File: 1816-truncate-sentence/1816-truncate-sentence.java
class Solution {
    public String truncateSentence(String s, int k) {
        String[] test = s.split(" ");
//         ["Hello" "how" "are" "you" "Contestant"] k = 6
        
        
        String result = "";
        int m = Math.min(k, test.length);
        for (int i = 0; i < m;i ++){
            result += test[i];
            if (i < m - 1) {
                result += " ";
            }
        }
        
        return result;
    }
}

File: 1816-truncate-sentence/1816-truncate-sentence.swift
class Solution {
    
    func truncateSentence(_ s: String, _ k: Int) -> String {
        let spl = s.split(separator: " ")
        var result = ""
        for i in 0..<k {
            result += spl[i]
            if i != k - 1 {
                result += " "
            }
        }
        return result
    }
}
